src/main/java/org/tron/common/overlay/discover/node/statistics/Reputation.java

```diff
@@ -0,0 +1,155 @@
+package org.tron.common.overlay.discover.node.statistics;
+
+import static java.lang.Math.min;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.tron.protos.Protocol.ReasonCode;
+
+public class Reputation {
+
+  public abstract class Score<T> implements Comparable<Score> {
+
+    T t;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "t". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1284&issues=AWUh7dqNGvJTV_KzqSfP&open=AWUh7dqNGvJTV_KzqSfP)

[ISSUE_KEY]: AWUh7dqNGvJTV_KzqSfP